src/sql/util.py

```diff
@@ -494,3 +494,16 @@ def get_default_configs(sql):
     del default_configs["parent"]
     del default_configs["config"]
     return default_configs
+
+
+def _are_numeric_values(*values):
+    return all([isinstance(value, (int, float)) for value in values])
+
+
+def validate_mutually_exclusive_args(arg_names, args):
```

Comment:
Please add docstring explaining `arg_names` and `args`

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/789/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R504-R514)